### DISCUSSION TITLE
JS for data-hook="cart_item_delete"

### DISCUSSION THREAD

**Discussion by ViciousMagician** [solidusio/solidus#5807] [Troubleshooting] [answered]
Can anyone point me to where the JS is that handles this action?

app/views/spree/orders/_line_item.html.erb. 

Were trying to convert to jsbundling for Rails 7 and this action is no longer working.  

Currently at 4.3.3

**Comment by jarednorman** [comment]
Starter frontend or legacy frontend?

**Reply by ViciousMagician** [reply]
were using gem "solidus_frontend"

**Reply by jarednorman** [reply, marked as answer]
Okay, that's the legacy frontend. Here's the cart code: https://github.com/solidusio/solidus_frontend/blob/215fab98ac1bb3a7f527633448551e8808cb8917/app/assets/javascripts/spree/frontend/cart.js
